custom/src/main/java/com/splunk/opentelemetry/SplunkConfiguration.java

```diff
@@ -43,6 +43,15 @@
     config.put("otel.instrumentation.spring-batch.enabled", "true");
     config.put("otel.instrumentation.spring-batch.item.enabled", "true");
 
+    // metrics exporter sometimes logs "Broken pipe (Write failed)" at INFO; usually in docker-based
+    // environments
+    // most likely docker resets long-running connections and the exporter has to retry, and it
+    // always succeeds after
+    // retrying and metrics are exported correctly
+    config.put(
+        "io.opentelemetry.javaagent.slf4j.simpleLogger.log.com.signalfx.shaded.apache.http.impl.execchain.RetryExec",
+        "WARN");
```

Comment:
can this still be overridden by a user in their own config?

Reply:
Yes - `PropertySource` only contains defaults that can be overridden using env vars/system properties.